schedule/yast/maintenance/offline_migration.yaml

```diff
@@ -0,0 +1,28 @@
+name: schedule/yast/maintenance/migration_offline.xml
```

Comment:
There is none split module loaded for patch_sle, maybe this works for 15SP4, but for 15SP3 we may need load modules such as [drop_unavailable_product_modules](https://openqa.suse.de/tests/13713563/modules/drop_unavailable_product_modules/steps/1/src) so the name of the yaml file couldn't be generic. Change it to offline_migration_15sp4 WDYT?

Reply:
changed.

Reply:
@lemon-suse  can you please take a look again?  thanks